db/migrate/20160829175601_add_content_filter_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddContentFilterToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :content_filter, :string
```

Comment:
1.  Not sure I understand. is the suggestion that there's a class for each type of filter, each class having one instance per user?
2. I thought a single serialized column might better anticipate a wider variety of future developments:
On Tue, Aug 2, 2016 at 1:36 AM Jason Hollinger <pellaea@gmail.com> wrote:
...
Again, I think it's worth looking ahead to other uses of this mechanism.  For example, instead of a boolean column in users for this one filter, it could be a serialized hash of arbitrary parameters and values.  For example it might read "{ has_images: yes }" for users who wish to hide imageless observations.  Or Adolf might like "{ has_images: yes, has_specimen: yes }".